.vuepress/components/Home.vue

```diff
@@ -0,0 +1,314 @@
+<template lang="pug">
+  div
+    .search__container
+      .search(@click="$emit('search', true)")
+        .search__icon
+          icon-search
+        .search__text Search
+    .h1 {{$frontmatter.title}}
+    .intro
+      .p__alt Learn how to use #[a(href="https://github.com/cosmos/cosmos-sdk" target="_blank" rel="noreferrer noopener") Cosmos SDK], the world’s most popular framework for building application-specific blockchains.
+    .sections__wrapper
+      .sections
+        router-link.sections__item(tag="a" :to="section.url" v-for="section in $frontmatter.sections")
+          .sections__item__tag(:style="{'--tag-text-color': `${tagColor[section.tag][0]}`, '--tag-background-color': `${tagColor[section.tag][1]}`}") {{section.tag}}
+          .sections__item__wrapper
+            .sections__item__title {{section.title}}
+            .sections__item__desc {{section.desc}}
+    .h2 Videos
+    .stack
+      a.stack__item(:href="item.url" target="_blank" rel="noreferrer noopener" v-for="item in $frontmatter.stack")
+        img(:src="item.imgSrc" alt="Image").stack__item__image
+        .stack__item__text
+          .stack__item__h1 {{item.title}}
+          .stack__item__p {{item.duration}}
+    tm-help-support
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      tagColor: {
+        beginner: [
+          '#5064FB',
+          '#FFFFFF',
+        ],
+        intermediate: [
+          '#FFFFFF',
+          '#5064FB',
+        ],
+        advanced: [
+          '#FFFFFF',
+          'rgba(0, 2, 36, 0.928)',
+        ],
+      },
+    }
+  }
+}
+</script>
+
+<style lang="stylus" scoped>
+a
+  color var(--color-link)
+  text-decoration none
+  cursor pointer
+
+.search {
+  display: flex;
+  align-items: center;
+  color: rgba(22, 25, 49, 0.65);
+  padding-top: 1rem;
+  width: calc(var(--aside-width) - 6rem);
+  cursor: pointer;
+  transition: color 0.15s ease-out;
+
+  &:hover {
+    color: var(--color-text, black);
+  }
+
+  &__container {
+    display: flex;
+    justify-content: flex-end;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+
+  &__icon {
+    width: 1.5rem;
+    height: 1.5rem;
+    fill: #aaa;
+    margin-right: 0.5rem;
+    transition: fill 0.15s ease-out;
+  }
+
+  &:hover &__icon {
+    fill: var(--color-text, black);
+  }
+}
+
+.intro {
+  width: 100%;
+  max-width: 800px;
+}
+
+.h1 {
+  font-size: 3rem;
+  font-weight: 700;
+  margin-bottom: 1.5rem;
+  line-height: 3.25rem;
+  letter-spacing: -0.02em;
+  padding-top: 2.5rem;
+}
+
+.h2 {
+  font-size: 2rem;
+  font-weight: 700;
+  margin-top: 4.5rem;
+  margin-bottom: 1rem;
+  line-height: 2.25rem;
+  letter-spacing: -0.01em;
+  padding: 1.5rem 0;
+}
+
+.p {
+  font-size: 1.5rem;
+  line-height: 2.25rem;
+
+  &__alt {
+    margin-top: 0.75rem;
+    margin-bottom: 2rem;
+    font-size: 1.25rem;
```

Comment:
Another failing on my part to update the Figma file with the newer type styles. Let's increase this to:
```
font-size 1.5rem
line-height 2.25rem
```
Then reduce to the sizes you have here @ `(max-width: 1136px)` instead – see the SDK docs homepage for reference.